src/core/trulens/core/database/sqlalchemy.py

```diff
@@ -378,7 +378,7 @@ def batch_insert_record(
                 self.orm.Record.parse(r, redact_keys=self.redact_keys)
                 for r in records
             ]
-            session.bulk_save_objects(records_list)
+            session.add_all(records_list)
```

Comment:
Whats difference between add_all and bulk_save objects?

Reply:
Functionally the same thing. [bulk_save_objects](https://docs.sqlalchemy.org/en/20/orm/session_api.html#sqlalchemy.orm.Session.bulk_save_objects) is the legacy API in SQLAlchemy 2.0 so updated this to use the `add_all` API.